src/layers/conv.jl

```diff
@@ -506,3 +506,44 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", AGGR2STR[l.aggr])
     print(io, ")")
 end
+
+
+"""
+    GINConv([graph, ] nn, eps, train_eps
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::R
+    train_eps::Bool
+end
+
+function GINConv(adj::AbstractMatrix, nn, eps, train_eps)
```

Comment:
```suggestion
function GINConv(fg::AbstractFeaturedGraph, nn, eps, train_eps)
```

The APIs have been changed by me and Carlo Lucibello. A layer accepts `FeaturedGraph` directly and don't need to support adjacency matrix.

Reply:
Fixed

Reply:
Done